src/commands/deploy.js

```diff
@@ -123,7 +127,9 @@ exports.handler = async (args = {}) => {
   }
 
   // create config if doesn't exist
-  const configPath = path.join(workdir, 'exoframe.json');
+  const configFilename = args.config || 'exoframe.json';
+  const configPath = path.join(workdir, configFilename);
```

Comment:
Just a reminder - as I've noted before, this won't be enough since server also picks up `exoframe.json`. The best way would probably be to rename the given config to `exoframe.json` (and ignore the original one if present) during packaging stage.